psh/test-ls-rootfs.py

```diff
@@ -21,8 +21,8 @@
 
 
 def assert_ls_pshcmds(p, psh_cmds):
-    # history, exit, clear and reset symlinks shouldn't be present in bin
-    psh_cmds = set(psh_cmds) - {'history', 'exit', 'clear', 'reset'}
+    # history, exit, clear, reset, export and unset symlinks shouldn't be present in bin
+    psh_cmds = set(psh_cmds) - {'history', 'exit', 'clear', 'reset', 'export', 'unset'}
 
     psh_cmd_pattern = CONTROL_CODE
```

Comment:
This should not be marked as part of your PR
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/b398561a-17cc-4511-bbae-aa76332f384f)